app/policies/user_policy.rb

```diff
@@ -2,4 +2,8 @@ class UserPolicy < ApplicationPolicy
   def destroy?
     user.admin?
   end
+
+  def index?
```

Comment:
It looks a little bit strange tbh. For me, `:index?` on UserPolicy means something along the lines `can given user watch index of users`? And in that case, what you want is `can given user watch admin panel`?
What I would like more is something like an `AdminPolicy`, but in this context, every action in the admin panel should be accessible only for admin, so you probably do not need pundit for it at all.
You can just use some simple before_action in the controller, which checks if user is an admin, and calls `user_not_authorized` otherwise.
Something like `before_action :user_not_authorize, if: -> { !current_user.admin? }`.